src/query_results.js

```diff
@@ -216,84 +216,84 @@ module.exports = class QueryResult {
+    debug(`Nodes with "is_set": ${JSON.stringify([...qNodeIDsWithIsSet])}`)
 
     // find a QNode having only one QEdge to use as the root node for tree traversal
-    let initialQueryEdgeID, initialQueryNodeIDToMatch;
-    toPairs(dataByEdge).some(([queryEdgeID, {connected_to, records}]) => {
-      const inputQueryNodeID = helper._getInputQueryNodeID(records[0]);
-      const outputQueryNodeID = helper._getOutputQueryNodeID(records[0]);
+    let initialQEdgeID, initialQNodeIDToMatch;
+    toPairs(recordsByQEdgeID).some(([queryEdgeID, {connected_to, records}]) => {
+      const inputQNodeID = helper._getInputQueryNodeID(records[0]);
+      const outputQNodeID = helper._getOutputQueryNodeID(records[0]);
 
       if (connected_to.length === 0) {
-        initialQueryEdgeID = queryEdgeID;
-        initialQueryNodeIDToMatch = inputQueryNodeID;
+        initialQEdgeID = queryEdgeID;
+        initialQNodeIDToMatch = inputQNodeID;
       } else {
         connected_to.some((c) => {
-          const nextEdge = dataByEdge[c];
-          const inputQueryNodeID1 = helper._getInputQueryNodeID(nextEdge.records[0]);
-          const outputQueryNodeID1 = helper._getOutputQueryNodeID(nextEdge.records[0]);
-          if (!initialQueryEdgeID) {
-            if ([inputQueryNodeID1, outputQueryNodeID1].indexOf(inputQueryNodeID) === -1) {
-              initialQueryEdgeID = queryEdgeID;
-              initialQueryNodeIDToMatch = inputQueryNodeID;
+          const nextEdge = recordsByQEdgeID[c];
+          const inputQNodeID_1 = helper._getInputQueryNodeID(nextEdge.records[0]);
+          const outputQNodeID_1 = helper._getOutputQueryNodeID(nextEdge.records[0]);
+          if (!initialQEdgeID) {
+            if ([inputQNodeID_1, outputQNodeID_1].indexOf(inputQNodeID) === -1) {
+              initialQEdgeID = queryEdgeID;
+              initialQNodeIDToMatch = inputQNodeID;
 
               // like calling break in a loop
               return true;
-            } else if ([outputQueryNodeID1, outputQueryNodeID1].indexOf(outputQueryNodeID) === -1) {
-              initialQueryEdgeID = queryEdgeID;
-              initialQueryNodeIDToMatch = outputQueryNodeID;
+            } else if ([outputQNodeID_1, outputQNodeID_1].indexOf(outputQNodeID) === -1) {
+              initialQEdgeID = queryEdgeID;
+              initialQNodeIDToMatch = outputQNodeID;
 
               // like calling break in a loop
               return true;
             }
           }
         });
 
-        if (initialQueryEdgeID) {
+        if (initialQEdgeID) {
           // like calling break in a loop
           return true;
         }
       }
     });
 
-    debug(`initialQueryEdgeID: ${initialQueryEdgeID}, initialQueryNodeIDToMatch: ${initialQueryNodeIDToMatch}`);
+    debug(`initialQEdgeID: ${initialQEdgeID}, initialQNodeIDToMatch: ${initialQNodeIDToMatch}`);
 
     // 'preresult' just means it has the data needed to assemble a result,
```

Comment:
Appears the new vocab uses `unconsolidatedResults` or `ucResults` instead of `preresult`? If that's the choice, we'd want to make that change in the comments too, like here.

Reply:
Also, if we use some form of `ucResult` anywhere, I'd suggest using it everywhere instead of `unconsolidatedResult` in order to be consistent.

Reply:
I must have missed some spots. I should also note that I used `unconsolidatedResult` (shortform `ucResult`) instead of `preresult` mostly due to the BTE Vocab sheet stating this proposal - I had assumed this was more agreed upon -- should this be discussed further? This part of the code is less familiar to me so I'd defer to your preference on what best reflects the concept's purpose etc

Reply:
Even though I originated the term `preresult`, I actually don't really like it. :) I just didn't think of anything better.

Here, I'm just pushing for consistency.

Reply:
I get the sense that this work https://github.com/biothings/bte_trapi_query_graph_handler/pull/88#issuecomment-1026413433 from @ariutta was not used by @tokebe when Jackson went through the whole package....do we need to resolve differences between them?